### PR TITLE
🎉  release prep

### PR DESCRIPTION
**Pull request by dmiller15**
## Description

- Port over the create somatic panel of normals workflow. Miguel and I decided we just want to have everything here so we won't end up having to maintain it in two places. Nothing has been changed so it should be good as is.
- Copied over documentation for the two GATK workflows that I had previously forgotten
- Added some new tags to the somatic workflow for the new pieces of the workflow
- Bumped version to 4.0.0

## Type of change

- [x] New feature (non-breaking change which adds functionality)
- [x] Documentation

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Shouldn't be anything new to test

**Test Configuration**:
* Environment:
* Test files:

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings
- [x] I have committed any related changes to the PR

### CODE REVIEW

File: workflow/kfdrc-somatic-variant-workflow.cwl [hunk 1]
```diff
@@ -936,18 +936,22 @@ hints:
...
 - CONTROLFREEC
 - CRAM
+- FUNCOTATOR
+- GATK
 - INDEL
 - LANCET
 - MAF
 - MANTA
 - MUTECT2
+- PON
```
Comment (migbro) [inline workflow/kfdrc-somatic-variant-workflow.cwl:946]: Not sure you need this tag as I think this WF does not actually run PON, it just exists within the repo overall

**Review by sickler-alex — APPROVED** [review]

File: workflow/kfdrc-somatic-variant-workflow.cwl (hunk 1 above)
Comment (dmiller15) [reply workflow/kfdrc-somatic-variant-workflow.cwl:946]: The panel is an input.

Comment (dmiller15) [reply workflow/kfdrc-somatic-variant-workflow.cwl]: Ok removed.

File: workflow/kfdrc_gatk_create_cnv_pon_wf.cwl [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+'sbg:publisher': KFDRC
+'sbg:categories':
+- BAM
+- CNV
+- COUNTS
+- CRAM
+- HDF5
+- PON
+'sbg:links':
+- id: 'https://github.com/kids-first/kf-somatic-workflow/releases/tag/v0.2.0'
```
[the remark this answers is not in the document]
Comment (dmiller15) [reply workflow/kfdrc_gatk_create_cnv_pon_wf.cwl]: Bumped to 4.0.0

**Review by migbro — APPROVED** [review]
Looks good, thanks!

**Pull request merged**